src/pkcs11/framework-pkcs15.c

```diff
@@ -3879,8 +3879,9 @@ pkcs15_pubkey_get_attribute(struct sc_pkcs11_session *session, void *object, CK_
 		break;
 	case CKA_TOKEN:
 	case CKA_SENSITIVE:
```

Comment:
Before make this change, the question that should be asked, is why is OpenSC returning the CKA_SENSITIVE attribute for a public key, as that is not a valid attribute for a public key? 


When was this added, and why?